user/applications/tinker/src/burnin_test.cpp

```diff
@@ -36,6 +40,67 @@ extern uintptr_t platform_system_part1_flash_start;
 
 namespace particle {
 
+namespace detail {
+
+static const uint32_t softCrc32Tab[] = {
```

Comment:
Not sure if its better to copy/paste this here, or include in separate file?

Reply:
You can just include the one from tests probably or move it to a common location (e.g. `services`)